app/controllers/ops_controller/settings/rhn.rb

```diff
@@ -106,12 +106,13 @@ def rhn_save_subscription
     db          = MiqDatabase.first
     credentials = rhn_credentials_from_edit
 
+    db.update_repo_name = credentials[:update_repo_name]
+
     db.update_attributes(
       credentials.slice(
         :registration_type,
         :registration_server,
-        :registration_http_proxy_server,
-        :update_repo_name
+        :registration_http_proxy_server
```

Comment:
This is done. Not sure how to indicate just one part of a review is completed as the "outdated diff" stuff doesn't seem to apply to parts of a review.